android/filamat-android/src/main/java/com/google/android/filament/filamat/MaterialBuilder.java

```diff
@@ -98,7 +98,15 @@
         UV0,                    // texture coordinates (float2)
         UV1,                    // texture coordinates (float2)
         BONE_INDICES,           // indices of 4 bones (uvec4)
-        BONE_WEIGHTS            // weights of the 4 bones (normalized float4)
+        BONE_WEIGHTS,           // weights of the 4 bones (normalized float4)
+        CUSTOM0,
```

Comment:
Do we need to expose so many right away? What does this mean for blend shapes? We're only going to do it in glTF through those custom attributes?

Reply:
Morphing will leverage these CUSTOM attributes, it will not add to enum.

Unfortunately we need a total of 8 custom attributes (in addition to the existing "base" position and tangent) for minimal-but-sufficient support of morphing in the glTF spec.

Morphing is the motivating use case, but they can be used for other things as well.

By the way, we should perhaps consider removing vertex color now that we have this... but that can be done in the future.